build.gradle

```diff
@@ -1,13 +1,18 @@
+apply plugin: "de.fayard.buildSrcVersions"
+
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.3.72"
     repositories {
         google()
         jcenter()
+        gradlePluginPortal()
+        maven { url 'https://jitpack.io' }
```

Comment:
What are you using jitpack for?

Reply:
Oh I think I added that one for trying out another dependency update solution and I forgot to remove it. Nice catch! 
Will delete in #3 if that's ok?